packages/select/src/FzSelect.vue

```diff
@@ -179,15 +199,38 @@ const evaluateProps = () => {
 
 async function calculateContainerWidth() {
   await nextTick();
-  if (!opener.value) containerWidth.value = "auto";
-  else containerWidth.value = `${opener.value.clientWidth}px`;
+  if (!safeOpener.value) containerWidth.value = "auto";
+  else containerWidth.value = `${safeOpener.value.clientWidth}px`;
 }
 
 function calculateTopPosition() {
-  if (!opener.value || !openerContainer.value) return;
+  if (!safeOpener.value || !openerContainer.value) return;
   const { top, height } =
     openerContainer.value.parentElement!.parentElement!.getBoundingClientRect();
   containerTopPosition.value = top + height;
 }
+
+function addScrollListener() {
+  containerRef.value?.addEventListener("scroll", handleScroll);
+}
+
+function handleScroll() {
+  const container = containerRef.value!;
```

Comment:
per ora possiamo anche tenere così, ma per infinite scroll suggerirei di rifattorizzare utilizzando intersection observer. Ancora meglio forse creare un composable apposito forse?

Reply:
Non ho conoscenza su questo oggetto ma mi informo. Ci starebbe anche scorporare la logica in un composables in modo da utilizzarlo da altre parti (actionlist fra tutte)